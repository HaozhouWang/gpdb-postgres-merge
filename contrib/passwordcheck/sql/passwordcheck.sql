LOAD 'passwordcheck';
<<<<<<< HEAD
=======

>>>>>>> 9e1c9f959422192bbe1b842a2a1ffaf76b080196
CREATE USER regress_user1;

-- ok
ALTER USER regress_user1 PASSWORD 'a_nice_long_password';

-- error: too short
ALTER USER regress_user1 PASSWORD 'tooshrt';

-- error: contains user name
ALTER USER regress_user1 PASSWORD 'xyzregress_user1';

-- error: contains only letters
ALTER USER regress_user1 PASSWORD 'alessnicelongpassword';

<<<<<<< HEAD
=======
-- encrypted ok (password is "secret")
ALTER USER regress_user1 PASSWORD 'md51a44d829a20a23eac686d9f0d258af13';

-- error: password is user name
ALTER USER regress_user1 PASSWORD 'md5e589150ae7d28f93333afae92b36ef48';

>>>>>>> 9e1c9f959422192bbe1b842a2a1ffaf76b080196
DROP USER regress_user1;
