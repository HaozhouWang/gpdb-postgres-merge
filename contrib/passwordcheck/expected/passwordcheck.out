LOAD 'passwordcheck';
<<<<<<< HEAD
LOAD
CREATE USER regress_user1;
psql:passwordcheck.sql:2: NOTICE:  resource queue required -- using default resource queue "pg_default"
CREATE ROLE
-- ok
ALTER USER regress_user1 PASSWORD 'a_nice_long_password';
ALTER ROLE
-- error: too short
ALTER USER regress_user1 PASSWORD 'tooshrt';
psql:passwordcheck.sql:8: ERROR:  password is too short
-- error: contains user name
ALTER USER regress_user1 PASSWORD 'xyzregress_user1';
psql:passwordcheck.sql:11: ERROR:  password must not contain user name
-- error: contains only letters
ALTER USER regress_user1 PASSWORD 'alessnicelongpassword';
psql:passwordcheck.sql:14: ERROR:  password must contain both letters and nonletters
DROP USER regress_user1;
DROP ROLE
=======
CREATE USER regress_user1;
-- ok
ALTER USER regress_user1 PASSWORD 'a_nice_long_password';
-- error: too short
ALTER USER regress_user1 PASSWORD 'tooshrt';
ERROR:  password is too short
-- error: contains user name
ALTER USER regress_user1 PASSWORD 'xyzregress_user1';
ERROR:  password must not contain user name
-- error: contains only letters
ALTER USER regress_user1 PASSWORD 'alessnicelongpassword';
ERROR:  password must contain both letters and nonletters
-- encrypted ok (password is "secret")
ALTER USER regress_user1 PASSWORD 'md51a44d829a20a23eac686d9f0d258af13';
-- error: password is user name
ALTER USER regress_user1 PASSWORD 'md5e589150ae7d28f93333afae92b36ef48';
ERROR:  password must not equal user name
DROP USER regress_user1;
>>>>>>> 9e1c9f959422192bbe1b842a2a1ffaf76b080196
